compiler/dyno/include/chpl/framework/Context.h

```diff
@@ -247,6 +255,22 @@ class Context {
   Context();
   ~Context();
 
+  /**
+    Store a string corresponding to CHPL_HOME in the context. The context does
+    not take ownership of the string, so the argument should live at least as
+    long as the context itself.
```

Comment:
Can it just be a `std::string` so we don't have to worry so much? Why wouldn't you copy it?